src/main/java/org/ppijerman/ppijidentitybackend/server/dto/Person.java

```diff
@@ -0,0 +1,66 @@
+package org.ppijerman.ppijidentitybackend.server.dto;
+
+import lombok.Data;
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.List;
+import java.util.UUID;
+
+@Entity
+@Data
+@Table(name="PERSON", schema="CENSUS")
+public class Person {
+
+    @Id
+    @Column(name="person_id", columnDefinition = "uuid")
+    private UUID personId;
+
+    @Column(name="name", columnDefinition="VARCHAR(50)", length = 50, nullable=false)
```

Comment:
I just updated to 100 just in case someone was brave enough to name their kid with >50 characters